tests/plugins/test_import_image.py

```diff
@@ -304,11 +325,28 @@ def test_import_image(tmpdir, build_process_failed, namespace, monkeypatch, reac
         (flexmock(OSBS)
          .should_receive('ensure_image_stream_tag')
          .times(DEFAULT_TAGS_AMOUNT))
-        (flexmock(OSBS)
-         .should_receive('import_image')
-         .once()
-         .with_args(TEST_IMAGESTREAM)
-         .and_return(True))
+
+        if not import_image_with_tags:
```

Comment:
Small nit: when using `if not` with an `else` clause, consider omitting `not` and switching around the clauses.

Reply:
changed